### ISSUE TITLE
support for sublime text 3

### ISSUE THREAD

**Issue by zeroxme** [wbond/sublime_prefixr#21]
is there going to be any support for sublime 3 ?

thanks

**Comment by CiaranDeGroot** [comment]
Ugh, I hope so. AutoPrefixr doesn't work either, so there's no alternative... Hopefully he'll patch it some time soon.

**Comment by schmod** [comment]
I believe that this is related to the way that ST3 imports modules.  I'll take a look at it and patch it up in my fork if I can.  This (and the underlying prefixr API)  both seem to have been abandoned by their maintainers.  

I'm not sure if there are any other python libraries that will automatically add CSS prefixes... it may be worth investigating, given that prefixr is more or less dead.

**Comment by schmod** [comment]
https://github.com/myfreeweb/cssprefixer might be a good alternative to prefixr.  Runs locally, seems to be actively maintained...

**Comment by schmod** [comment]
This is fixed in https://github.com/schmod/sublime_prefixr/commit/620362b2bd363d1beadf144d4653938d979fa74f

Feel free to clone my repo to your `Packages` directory, or poke @wbond to merge my PR.

**Comment by CiaranDeGroot** [comment]
@schmod Aww dude, thanks a ton :D Your fork works beautifully.

**Comment by zeroxme** [comment]
@schmod thanks a lot ^_^ ... maybe a pull request would be great

**Comment by CiaranDeGroot** [comment]
@izer0x It doesn't seem like the wbond is really maintaining this anymore, so I doubt he'll accept...

**Comment by schmod** [comment]
I already have a PR (#19) open from a few months ago, and forgot to branch
for the latest PR. If @wbond merges #19, it'll fix a few bugs and also
include the fixes to support ST3.

**Comment by CiaranDeGroot** [comment]
@schmod Yeah, I noticed that - that's why I assumed he's not really bothering to merge anymore. Also, the last commit was two years back and there's one contributor...

**Comment by wbond** [comment]
It has mostly been an issue of time. I should hopefully have things updated for ST3 as soon PC 2.0 is beta.

**Comment by gsingh93** [comment]
Since PC 2.0 is out, I'm just going to bump this :)

**Comment by eneifert** [comment]
+1 for sublime text 3 support :)

**Comment by alexilyaev** [comment]
Ahm, any ETA on this one?

**Comment by AndrejTelle** [comment]
+1 for sublime text 3 support as well :) pls

**Comment by arudmin** [comment]
+1 I'm waiting for updates for ST3

**Comment by clemensmeyer** [comment]
+1 would love this for ST3 as well! :)

**Comment by alexilyaev** [comment]
AutoPrefixer works with ST3 and does a good job:
https://github.com/sindresorhus/sublime-autoprefixer

**Comment by clemensmeyer** [comment]
@alexilyaev Thanks! Just tried it out, it works like a charm.
